### PR TITLE
reduce cardinality of path label

### PR DESCRIPTION
**Pull request by mollylogue**

### CODE REVIEW

File: internal/actor/http.go [hunk 1]
```diff
@@ -134,3 +135,13 @@ func HTTPMiddleware(next http.Handler) http.Handler {
...
 	})
 }
+
+func getURLPath(urlPath string) string {
+	if strings.Contains(urlPath, "/.internal/git/github.com/") {
+		return "/.internal/git/github.com/"
+	}
+	if strings.Contains(urlPath, "/git/github.com/") {
+		return "/git/github.com/"
+	}
```
Comment (bobheadxi) [inline internal/actor/http.go:145]: Might want to make this code-host-agnostic, i.e. by grouping everything that is `/.internal/git` and `/git`

File: internal/actor/http.go [hunk 2]
```diff
@@ -64,7 +65,7 @@ func (t *HTTPTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 
 	actor := FromContext(req.Context())
-	path := req.URL.Path
+	path := getURLPath(req.URL.Path)
```
Comment (bobheadxi) [inline internal/actor/http.go:68]:
```suggestion
	path := getCondensedURLPath(req.URL.Path)
```

**Review by bobheadxi — APPROVED** [review]
lgtm in principle, left a recommendation

File: internal/actor/http.go (hunk 1 above)
Comment (mollylogue) [reply internal/actor/http.go:145]: 👍 great idea.

File: internal/actor/http.go [hunk 3]
```diff
@@ -134,3 +135,13 @@ func HTTPMiddleware(next http.Handler) http.Handler {
...
 	})
 }
+
+func getCondensedURLPath(urlPath string) string {
+	if strings.Contains(urlPath, "/.internal/git/") {
+		return "/.internal/git/"
+	}
+	if strings.Contains(urlPath, "/git/") {
+		return "/git/"
+	}
```
Comment (bobheadxi) [inline internal/actor/http.go:145]: one last nit: 

```suggestion
	if strings. HasPrefix(urlPath, "/.internal/git/") {
		return "/.internal/git/..."
	}
	if strings.HasPrefix(urlPath, "/git/") {
		return "/git/..."
	}
```

Comment (bobheadxi) [reply internal/actor/http.go:145]: This is assuming the paths do actually start like this!

Comment (mollylogue) [reply internal/actor/http.go:145]: Oh, good point. 😅 my brain is a bit done for the day

**Pull request merged**